src/monda/tests/sorad_retrieval_qc_plot_test.py

```diff
@@ -45,12 +46,19 @@
 ------------------------------------------------------------------------------
 
 Tom Jordan - tjor@pml.ac.uk - Feb 2022
+                            - Spring 2024
 """
 
 import sys
 import os
 import numpy as np
-from monda.sorad import access, plots, qc
+#from monda.sorad import access, plots, qc # note QC import does not work via monda (also issues in ADIAS)
+sys.path.append('/users/rsg/tjor/Sorad_mondademos/MONDA/src/monda/sorad')  # for 2024 changes, I have used a local import as
```

Comment:
remove hardcoded paths

Reply:
ok - I will now change the imports to how it should work when the pypi issue is fixed